src/js/components/Chart/__tests__/Chart-test.js

```diff
@@ -161,6 +178,17 @@ describe('Chart', () => {
     expect(tree).toMatchSnapshot();
   });
 
+  test('value style', () => {
+    const component = renderer.create(
+      <Grommet>
+        <Chart type="point" point="circle" values={STYLED_VALUES} />
+        <Chart type="bar" values={STYLED_VALUES} />
```

Comment:
Could 'line' use the styled values as well? I haven't seen it either on the storybook or the test file.

Reply:
No. I've indicated this in the documentation updates. Since line and area type charts don't have discrete graphic elements for each value, you can't.